### PR TITLE
update some of the readme with additional links and updates

### PR DESCRIPTION
**Pull request by phlptp**
### Summary

If merged this pull request will address #1263 and update the readme to refer to more examples and organize the readme a little better.  

### Proposed changes

- update the PR template to remove a line so it fits better on common screen sizes
- add example links in the examples folder readme
- add doc download links to the main readme and shift the install section down on the page.

### CODE REVIEW

**Comment by phlptp** [comment]
Some of the failing builds are the result of updated docker images, and the issues are fixed in develop

File: README.md [hunk 1]
```diff
@@ -112,13 +88,45 @@ A [Tutorial](https://github.com/GMLC-TDC/pesgm-2019-helics-tutorial) was prepare
 
 Our [ReadTheDocs](https://helics.readthedocs.io/en/latest/index.html) site provides a set of documentation including a set of introductory [examples](https://helics.readthedocs.io/en/latest/introduction/index.html), a [developers guide](https://helics.readthedocs.io/en/latest/developer-guide/index.html), complete Doxygen generated [API documentation](https://helics.readthedocs.io/en/latest/doxygen/annotated.html), and more.  A few more questions and answers are available on the [Wiki](https://github.com/GMLC-TDC/HELICS/wiki).
 
+### Documentation downloads
+-   [PDF Version](https://docs.helics.org/_/downloads/en/latest/pdf/)
+-   [EPUB docs](https://docs.helics.org/_/downloads/en/latest/epub/)
+-   [HTML Zip file](https://docs.helics.org/_/downloads/en/latest/htmlzip/)
```
Comment (nightlark) [inline README.md:94]: Reorder, PDF and HTML zip are likely better choices than EPUB for most users.
```suggestion
-   [PDF](https://docs.helics.org/_/downloads/en/latest/pdf/)
-   [HTML Zip file](https://docs.helics.org/_/downloads/en/latest/htmlzip/)
-   [EPUB](https://docs.helics.org/_/downloads/en/latest/epub/)
```

File: examples/README.md [hunk 2]
```diff
@@ -1,28 +1,40 @@
-# HELICS example
+# HELICS examples
```
Comment (nightlark) [inline examples/README.md:1]:
```suggestion
# HELICS Examples
```

File: examples/README.md [hunk 3]
```diff
@@ -1,28 +1,40 @@
...
 
-### Running a simple example
+## More Examples
 
-1. Open three terminal windows
-2. Run the following in the first window
+Please see the [HELICS Examples Repo](https://github.com/GMLC-TDC/HELICS-Examples) for many more examples than are available in the HELICS repository
+This includes examples in many of the interface languages.
+
+Good places to start includes
```
Comment (nightlark) [inline examples/README.md:8]:
```suggestion
Good places to start include:
```

File: examples/README.md [hunk 4]
```diff
@@ -1,28 +1,40 @@
...
+
+[C Example](https://github.com/GMLC-TDC/HELICS-Examples/tree/master/c)
+[Python Examples](https://github.com/GMLC-TDC/HELICS-Examples/tree/master/python/pi-exchange)
```
Comment (nightlark) [inline examples/README.md:11]:
```suggestion
[Python Example](https://github.com/GMLC-TDC/HELICS-Examples/tree/master/python/pi-exchange)
```

**Pull request merged**